features/change_files_spec.rb

```diff
@@ -22,6 +22,7 @@ class Class1
                                  ./spec/class1_spec.rb[1:1:3:1]
```

Comment:
rework it to expect really necessary examples. all over examples should be ignored (sub array matching)